node/abTest/commands/status.ts

```
@@ -6,9 +6,12 @@ const bucket = 'ABTest'
 const fileName = 'currentABTest.json'
 
 export async function ABTestStatus(ctx: ColossusContext): Promise<TestResult[]> {
-  const { vtex: { account }, resources: { vbase } } = ctx
+  const { vtex: { account }, resources: { router, vbase } } = ctx
 
   try {
+    if(await router.getWorkspaces(account) === null) {
+      throw new Error('Test not initialized')
```

Can you return a specific error status code here, like a `400` or a `404`?